packages/flutter/lib/src/material/material.dart

```diff
@@ -217,9 +220,31 @@ class Material extends StatefulWidget {
   /// Must be opaque. To create a transparent piece of material, use
   /// [MaterialType.transparency].
   ///
+  /// If inside a dark theme and [applyDarkThemeElevationOverlay] is [true],
+  /// a semi-transparent white will be composited on top this color to
+  /// indicated the elevation.
+  ///
   /// By default, the color is derived from the [type] of material.
   final Color color;
 
+  /// Apply a semi-transparent white overlay on top of [color] for dark themes.
+  ///
+  /// If [true] a semi-transparent white overlay will be applied to the [color]
+  /// property when the surrounding theme's brightness is dark. The transparency
+  /// is based off of [elevation] as per the Material Dark theme specification.
+  ///
+  /// If [false] the [color] will be used unmodified.
+  ///
+  /// If [null] then it will use the value of the surrounding
+  /// [ThemeData.applyDarkThemeElevationOverlay] which defaults to [false].
+  ///
+  /// See also:
+  ///   * [Material.color]
```

Comment:
Usually we provide a brief explanation for each see also.  The explanation makes it clear why the linked-to thing is relevant.